### ISSUE TITLE
[Feature Request] Support importing MusicBrains release ID in beets catalog

### ISSUE THREAD

**Issue by FlorentLM** [ampache/ampache#4054]
Hey there,
I installed Ampache for the first time, to try using a [beets ](https://beets.readthedocs.io/en/stable/) database as the main organisation method.
(the reason why I do this is because I do not want to edit the tags in my files (they are on a read-only filesystem), so I have beets doing hardlinks only with proper names, and have it keep its database populated with all the tag info)

Now in Ampache, importing the database mostly works. After import though, for some reason many albums were still missing the cover art (despite having a `cover.jpg` file present in each album folder, and having `album_art_preferred_filename = "cover.jpg"` in the config).

I noticed that imported albums all have the MusicBrainz _release group_ ID, but they do not have the _release_ ID, i.e. the ID to the actual album release which is what beets uses to identify all the info.

Manually adding the release ID to the albums in Ampache and then relaunching "Gather art" fixes the missing covers, I guess Ampache finds them more easily on MusicBrainz.
(Still no clue why the file on disk isn't picked up though - but that's for a separate issue)

Now I figure it would be pretty straightforward to add support for importing the release ID in the beet catalog, since we're already importing similar other fields, correct? (I know nothing about php, but I can give it a try)

The field in beets database is called `mb_albumid`

## Describe what you don't like about Ampache, or what it is missing

Support importing `mb_albumid` from a beets database, so albums in Ampache are matched against the actual release and not just the release group.

Also it seems the genres are not imported either? Basically I think there is a lot of possibility of interaction with beets that Ampache is missing out on right now :)

_Edited: for clarity and spelling, and to add the genres thing_

**Comment by lachlan-00** [comment]
Are there any other mbid's in there? i'm assuming it supports them all?

we look at these ones:
```
album artist id
album id
album status
album type
artist id
release group id
track id
```

**Comment by lachlan-00** [comment]
in the add section it could be checking with song mbid instead of the album id.

It does check release group id which is good but the mbid is probably wrong and that's gotta be an issue

![image](https://github.com/user-attachments/assets/4783138d-ca14-4010-a5e9-8e71809b6327)

**Comment by lachlan-00** [comment]
just checking the beets site and this looks like we've got a lot of fields wrong
https://docs.beets.io/en/stable/dev/library.html#album

something like this should cover all the fields
![image](https://github.com/user-attachments/assets/09a7b2c3-4787-417f-896d-35327b02186a)

**Comment by FlorentLM** [comment]
> and it also looks like it's using the album artist name instead of the ID which album::check uses
> 
> would you be able to print me out some data examples here so i can check this out?
> 
> I tried using beets but it confuses the crap out of me so i just use picard instead

Yes of course! I can do that later today, do you have any specific things in mind? :)

**Comment by lachlan-00** [comment]
no specifics but as long as you've got fully musicbrainz's/picarded music that i can see from your db i can work out where to map those fields to the ampache database fields.

there looks like there's a lot missing so we should try and cover everything. i'll try and work out where the file content is loaded and map that to a import array.

**Comment by FlorentLM** [comment]
I see there has been some changes recently, lmk if you need anything! :)
